src/templates/article.js

```diff
@@ -117,18 +131,13 @@ const Article = props => {
         <Layout>
             <BlogPostTitleArea
                 // TODO: Pull real image once available
-                // articleImage={meta['atf-image']}
-                articleImage={ARTICLE_PLACEHOLDER}
+                articleImage={meta['atf-image'] || ARTICLE_PLACEHOLDER}
```

Comment:
I added this back in as well at this point, since the writers can try to add their own images now and we can make sure this is supported properly.